common/flagdinjector/flagdinjector.go

```diff
@@ -24,12 +27,13 @@ const (
 	rootFileSyncMountPath = "/etc/flagd"
 )
 
+//go:generate moq -pkg fake -skip-ensure -out ./fake/flagdinjector_mock.go . IFlagdContainerInjector:MockFlagdContainerInjector
```

Comment:
if we're switching to `moq`, we can remove the `mockgen` command in the makefile that we used previously for generating the mocks: https://github.com/open-feature/open-feature-operator/blob/e3c8b4290be99d78b88ffef686531a38b97e61be/Makefile#L248

Reply:
we are actually not, this was probably a copy-paste error